### PR TITLE
Style Changes: button colors; dialog icon

### PR DESCRIPTION
**Pull request by marlo-longley**
### What's new?

This pull addresses discussion with @mitchellkeaney and @DiegoPino about finalizing ESIE project styles.

1. Made all buttons in the node form and webform the same dark green as requested.

2. Fixed mission icon in Jquery dialog. Webform uses Jquery UI. There is a documented conflict between Bootstrap and Jquery UI styles for this icon having to do with the order they are loaded. The safest, non-interfering option was this CSS fix. There is a JS fix too, involving `.noConflict()` but it was more general to all buttons, and I didn't want unintended results.

### CODE REVIEW

File: css/webform.css [hunk 1]
```diff
@@ -95,3 +90,20 @@
     text-decoration: underline;
     color: #eb8d00;
 }
+/* Because of the order of loading Bootstrap and Jquery UI, there is a known issue with the titlebar-close button styles */
+/* Rather than reorder JS loading introducing possible bugs, manually load the jquery-ui "X" icon (icon sheet downloaded from Jquery UI) */
+/* This overrides the Jquery UI styles loaded by webform module */
+.ui-dialog-titlebar-close {
```
Comment (DiegoPino) [inline css/webform.css:96]: @marlo-longley was this affecting every pop up or just the webform ones? Just for sanity, did you check the pop ups generated by a panorama tour?

Comment (marlo-longley) [reply css/webform.css:96]: @DiegoPino specifically that jquery UI css was being loaded by Webform module. 

Good point. Do you mean this popup?
Screen Shot 2020-01-28 at 1 41 09 PM

Comment (DiegoPino) [reply css/webform.css:96]: nope. Not that one (i see no popup there right?). The one that opens another object. Nevermind. I just removed a lot of code from the panorama builder to fix some of the issues, so you won't be able to add hotspots until tomorrow. Will merge as it is. Thanks

Comment (marlo-longley) [reply css/webform.css:96]: @DiegoPino I guess that is a tooltip. :)
The class '.ui-dialog-titlebar-close' should only apply to jquery dialog boxes.
https://github.com/jquery/jquery-ui/blob/0c860b0d92f9959f6747f8c02e9671eb2fc561aa/ui/widgets/dialog.js#L437
I will double check this once I can test in Panorama Builder. Thanks!

**Pull request merged**